config/initializers/sentry.rb

```diff
@@ -13,6 +13,17 @@ def filter_record_not_unique_exception_messages!(event, hint)
   end
 end
 
+def ignore_sidekiq_errors_if_retry(event)
+  job = event&.transaction&.context&.job
+  if event&.exception.is_a?(Faraday::Error) &&
+       (
+         job.is_a?(ActiveStorage::AnalyzeJob) ||
+           job.is_a?(ActiveStorage::PurgeJob)
+       ) && job.retry?
+    nil
```

Comment:
I don't think this'll quite work, it'll return `nil` from the `ignore_sidekiq_errors_if_retry` method, but later on where we use this method we don't look at the return value, and the crucial bit is that we return `nil` from the `before_send` callback. We probably want to change this method to return a boolean, and then do something like:

```
return nil if ignore_sidekiq_errors_if_retry(event)
```

Assuming you've decided that returning `true` from `ignore_sidekiq_errors_if_retry` means we want to ignore the event.